app/controllers/journeys/expenses/workplaceRunningCosts/workingFromHome/MoreThan25HoursController.scala

```diff
@@ -0,0 +1,91 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.journeys.expenses.workplaceRunningCosts.workingFromHome
+
+import controllers.actions.{DataRequiredAction, DataRetrievalAction, IdentifierAction}
+import forms.expenses.workplaceRunningCosts.workingFromHome.MoreThan25HoursFormProvider
+import models.common.{BusinessId, TaxYear}
+import models.database.UserAnswers
+import models.{Mode, NormalMode}
+import navigation.ExpensesNavigator
+import pages.expenses.workplaceRunningCosts.workingFromHome.MoreThan25HoursPage
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import services.SelfEmploymentService
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.journeys.expenses.workplaceRunningCosts.workingFromHome.MoreThan25HoursView
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.Try
+
+@Singleton
+class MoreThan25HoursController @Inject() (override val messagesApi: MessagesApi,
+                                           service: SelfEmploymentService,
+                                           navigator: ExpensesNavigator,
+                                           identify: IdentifierAction,
+                                           getData: DataRetrievalAction,
+                                           requireData: DataRequiredAction,
+                                           formProvider: MoreThan25HoursFormProvider,
+                                           val controllerComponents: MessagesControllerComponents,
+                                           view: MoreThan25HoursView)(implicit ec: ExecutionContext)
+    extends FrontendBaseController
+    with I18nSupport {
+
+  def onPageLoad(taxYear: TaxYear, businessId: BusinessId, mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData) {
+    implicit request =>
+      val form = request.userAnswers
+        .get(MoreThan25HoursPage, Some(businessId))
+        .fold(formProvider(request.userType))(formProvider(request.userType).fill)
+
+      Ok(view(form, mode, request.userType, taxYear, businessId))
+  }
+
+  def onSubmit(taxYear: TaxYear, businessId: BusinessId, mode: Mode): Action[AnyContent] = (identify andThen getData andThen requireData) async {
+    implicit request =>
+      def handleSuccess(userAnswers: UserAnswers, answer: Boolean): Future[Result] = {
+        val redirectMode = continueAsNormalModeIfPrevAnswerChanged(answer)
```

Comment:
continueAsNormalModeIfPrevAnswerChanged is something new, let's add few bespoke tests on top of the default set we have.

- Create `FakeExpensesTwoRoutesNavigator`:

```
class FakeExpensesTwoRoutesNavigator(desiredRoute: Call, checkDesiredRoute: Call) extends ExpensesNavigator {
  override def nextPage(page: Page, mode: Mode, userAnswers: UserAnswers, taxYear: TaxYear, businessId: BusinessId): Call =
    mode match {
      case NormalMode => desiredRoute
      case CheckMode  => checkDesiredRoute
    }
}
```

In `MoreThan25HoursControllerSpec` you can then:

```
  def checkOnwardRoute: Call        = routes.MoreThan25HoursController.onPageLoad(taxYear, businessId, CheckMode)

  override val bindings: List[Binding[_]] = List(
    bind[ExpensesNavigator].toInstance(new FakeExpensesTwoRoutesNavigator(onwardRoute, checkOnwardRoute))
  )

// add tests for all combinations:
// currentAnswer = false | true
// previousAnswer = None | Some(true) | Some(false)
```

Let's do it via Table 

````
Table(
   ("prevAnswer", "currAnswer", "expectedUrl")
  (None, true, ...)
  etc.
) 
```